examples/trajectory_point_interface.cpp

```diff
@@ -159,17 +161,27 @@ int main(int argc, char* argv[])
     std::vector<urcl::vector6d_t> points{ { -0.203, 0.263, 0.559, 0.68, -1.083, -2.076 },
                                           { -0.203, 0.463, 0.559, 0.68, -1.083, -2.076 } };
     std::vector<double> motion_durations{ 5.0, 5.0 };
+    std::vector<double> velocities{ 2.0, 2.3 };
+    std::vector<double> accelerations{ 2.5, 2.5 };
     std::vector<double> blend_radii{ 0.0, 0.0 };
 
     // Trajectory execution
     g_my_driver->writeTrajectoryControlMessage(urcl::control::TrajectoryControlMessage::TRAJECTORY_START,
-                                               points.size());
+                                               points.size() * 2);
     for (size_t i = 0; i < points.size(); i++)
     {
       // setting the cartesian parameter makes it interpret the 6d vector as a pose and use movel
       g_my_driver->writeTrajectoryPoint(points[i], true, motion_durations[i], blend_radii[i]);
     }
 
+    // Same motion, but parametrized with acceleration and velocity
+    motion_durations = { 0.0, 0.0 };
```

Comment:
Maybe we should have some different "writeTrajecotryPoint() function. Like one without the duration, as it is optional to define it. When we do not need to have a zero vector here